src/physlr-molecules.cc

```diff
@@ -482,4 +609,4 @@ main(int argc, char* argv[])
 		std::cerr << "in sec: " << omp_get_wtime() - sTime << std::endl;
 #endif
 	}
-}
+}
```

Comment:
```suggestion
}

```
Add an empty line here